docs/guides/support.md

```diff
@@ -29,22 +30,73 @@
 
 ## Overview
 
-Ongoing support for Carbon and previous version (which will be prioritized in
-that order)
+Ongoing support for the latest version Carbon and the previous version (which
+will be prioritized in that order)
+
+Whether you are new or a seasoned veteran, being a maintainer of the Carbon
+Design System can be overwhelming and can often lead to feelings of imposter
+syndrome. Whether you are triaging new issues in Github or responding to Slack
+threads in our support channels, here are some responses that can help get the
+conversation started in case you get stuck.
+
+- “Could you share more about you’ve already tried?”
+- “Can you share more about your use-case?”
+- “Can you add a repro (reproduction) using Stackblitz?”
+- “After searching through FILL IN RESOURCE HERE, I was able to find this and
```

Comment:
Just a suggestion: When saying ...FILL IN RESOURCE HERE, you could also put it in brackets like this [FILL IN RESOURCE HERE]